notification-support/src/test/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventListenerKafkaIntegrationTest.java

```diff
@@ -243,39 +249,31 @@ public static String getSubjectName(String topic, boolean isKey) {
   public NotificationEvent<Schema> getDummySchemaEvent(int event, EventType eventType, String source) {
     log.info("Emitting event {}", event);
     Schema schema = getDummySchema();
-    return NotificationEvent.<Schema>builder()
+    return NotificationEvent.<Schema> builder()
         .entity(schema)
         .source(source)
         .eventType(eventType)
         .build();
   }
 
   public static Schema getDummySchema() {
-    val name = Instant.now().toString();
```

Comment:
Why remove `lombok.val`? Strictly I guess we should use `var` being java 11

Reply:
because it adds nothing. All the values are Strings and all the others are identified as such.

Reply:
But you have yourself used Java's `var` elsewhere.

Reply:
Ok... but in this context we have a several Strings and a var for no reason.

Reply:
I'm not arguing that val/var add nothing. I'm saying this specific use of val  adds nothing, if anything it is more confusing since all the other variables are defined as Strings except this one (which is also a String)